yarn.lock

```diff
@@ -92,7 +92,7 @@
     ts-invariant "^0.4.4"
     tslib "^1.10.0"
 
-"@apollo/react-ssr@^3.1.3":
+"@apollo/react-ssr@3.1.3":
```

Comment:
Is this the correct yarn.lock file? I don't understand why this line would have changed?

Reply:
My bad! Fixed.